Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -247,18 +246,18 @@ void ConfigurePluginsServices()
             {
                 _logger.Debug("Configuring services in plugins:");
                 
-                foreach (var type in TypeHelper.GetPluginTypes())
+                foreach (var servicesConfigurationActivity in TypeHelper.GetAssemblyTypes<IServicesConfigurationActivity>())
                 {
-                    _logger.Debug($"Plugin {type.Name}");
+                    _logger.Debug($"Executing services configuration activity '{servicesConfigurationActivity.Name}'...");
 
                     try
                     {
-                        var plugin = (IPlugin) Activator.CreateInstance(type);
+                        var plugin = (IServicesConfigurationActivity) Activator.CreateInstance(servicesConfigurationActivity);
```

Comment:
Just saw this. Rename to `var activity = ` or `var configurationActivity = ` or similar.